version.json

```diff
@@ -0,0 +1,16 @@
+{
+  "version": "1.3.0-preview.1.build.{height}",
+  "publicReleaseRefSpec": [
+    "^refs/heads/master$", // we release out of master    
+    "^refs/tags/v\\d+\\.\\d+" // we also release tags starting with vN.N
```

Comment:
I use `heads` for the second line most of the time, FWIW. That way release branches can ship without tagging just like master can.